src/situatedSIPP/learningAlgorithms/DijkstraLearning.cpp

```diff
@@ -1,4 +1,3 @@
-#pragma once
 #include "./DijkstraLearning.hpp"
 #include "set"
```

Comment:
we should use angle bracket for standard library include and double quatation include for local source file include, eg #include \<set\>. And for style consistentcy, let's always inlude stardard library at top, and put our souce file after, eg put line 2 after.